src/Components/Cart.js

```diff
@@ -0,0 +1,23 @@
+import { useData } from "../data-context";
+import { ProductCard } from "./ProductCard"
+
+export function Cart() {
+    const{ itemsInCart } = useData();
+    return(
+        <>
+            {(itemsInCart.lentgh === 0) ?
+                (
+                    <div>
+                        <h1>Cart is Empty</h1>
+                    </div>
+                )
+            :
+                (
+                    <div className="App" style={{ display: "flex", flexWrap: "wrap" }}>
+                        {itemsInCart.map( (item) => (<ProductCard product={item} />) )}
```

Comment:
hey rohit, i think you should provide a `key`  in `<ProductCard key={item.id} product={item} />` because console will give error if you don't provide it.